test/swarming/bot-scripts/video.py

```diff
@@ -84,6 +86,45 @@ def main():
         '-out', videofile,
         gfxtrace
     ]
+    p = botutil.runcmd(cmd)
+    
+    if p.returncode != 0:
+        return p.returncode
+
+
+    cmd = [
+        gapit,
+        'commands',
+        '-onlyexecuteddraws',
+        gfxtrace
+    ]
+    p = botutil.runcapture(cmd)
+
+    if p.returncode != 0:
+        return 1
+
+    commands = []
+    for line in p.stdout.splitlines():
+        if line.startswith('['):
+            commands.append(re.search(r'^\[.*\]', line).group(0).replace(' ', ', '))
```

Comment:
Anytime you use regexps, I highly encourage to document what they are expected to match:

```suggestion
        # Lines with draw calls commands looks like: "[1 2 3 4] vkCmdDraw..."
        # Here we select those lines, extract the command ID and add commas
        # between the indexes to get: "[1, 2, 3, 4]"
        if line.startswith('['):
            commands.append(re.search(r'^\[.*\]', line).group(0).replace(' ', ', '))
```